onnxruntime/test/providers/cpu/math/inverse_test.cc

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "gtest/gtest.h"
+#include "test/providers/provider_test_utils.h"
+#include "core/util/math.h"
+
+namespace onnxruntime {
+namespace test {
+
```

Comment:
Needs some tests where there are batches, preferably including where multiple dimensions provide the number of batches (e.g. rank 4 input).